.github/workflows/deploy-staging.yml

```diff
@@ -13,15 +14,14 @@ jobs:
     uses: ./.github/workflows/linting.yml
     secrets: inherit
 
-  # deploy to staging
   scan-dev:
-    name: Zap scan of the dev site
+    name: Zap Scan
     uses: ./.github/workflows/zap-scan.yml
     with:
       url: "https://fac-dev.app.cloud.gov/"
 
   deploy-infrastructure-staging:
-    name: Deploy infrastructure (staging)
+    name: Terraform Apply
```

Comment:
Same as elsewhere: "Terraform Apply" probably means less to people unfamiliar with what Terraform is/does than "Deploy infrastructure", which one can more intuitively figure out is "all the framing that goes around the app, which should get updated first".